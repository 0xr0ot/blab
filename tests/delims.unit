S = 'foo' "bar" '"baz"' "'quux'" '\n'
    '"rat\"\'s' "'buttock's\'\"\n"
    '\a\n'
