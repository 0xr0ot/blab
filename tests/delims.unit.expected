foobar"baz"'quux'
"rat"'s'buttock's'"
\a
